compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -588,6 +625,35 @@ fn import_relay_resolver_function_type(
         &PathBuf::from(resolver_metadata.import_path.lookup()),
     );
 
+    let context_import = if resolver_metadata.live {
```

Comment:
Even non-live resolvers should be able to read from context I think. I could imagine a purely derived resolver which wants to look into context to see something like the user's configured language or if they are in dark mode.